packages/block-library/src/embed/edit.native.js

```diff
@@ -282,11 +289,13 @@ const EmbedEdit = ( props ) => {
 			) }
 			<EmbedBottomSheet
 				value={ url }
-				isVisible={ isEditingURL }
-				onClose={ () => setIsEditingURL( false ) }
+				isVisible={ showEmbedBottomSheet }
+				onClose={ () => {
+					setShowEmbedBottomSheet( false );
+				} }
```

Comment:
The `onClose` prop should only be used for controlling the bottom sheet's visibility.